storage/storage-samples/storage-rv-sample/src/main/java/org/fido/iot/storage/RvsDbManager.java

```diff
@@ -61,6 +61,22 @@ public void createTables(DataSource dataSource) {
 
       stmt.executeUpdate(sql);
 
+      sql = "CREATE TABLE IF NOT EXISTS "
```

Comment:
do we allow for persistent storage of allow/deny lists? Or are these tables re-created each time?

Reply:
The tables (allow and deny list) are created each time the application is started and it remains persistent until we stop the application